### PR TITLE
Exclude `.git` for default `fd` command

### PR DESCRIPTION
**Pull request by matthewgrossman**
Fixes https://github.com/nvim-telescope/telescope-frecency.nvim/issues/211

### CODE REVIEW

File: doc/telescope-frecency.txt [hunk 1]
```diff
@@ -584,8 +584,8 @@ Type: `"LUA"|string[]|nil`
 With the default value: `nil`, it uses these ways below to make entries for
 workspace files. It tries in order until it works successfully.
 
-1. `fdfind -Htf`
-2. `fd -Htf`
+1. `fdfind -Htf --exclude .git`
+2. `fd -Htf --exclude .git`
```
Comment (delphinus) [inline doc/telescope-frecency.txt:588]:
```suggestion
1. `fdfind -Htf -E .git`
2. `fd -Htf -E .git`
```

File: lua/frecency/finder.lua [hunk 2]
```diff
@@ -83,7 +83,7 @@ function Finder:start(epoch)
   if config.workspace_scan_cmd ~= "LUA" and self.need_scan_dir then
     ---@type string[][]
     local cmds = config.workspace_scan_cmd and { config.workspace_scan_cmd }
-      or { { "fdfind", "-Htf" }, { "fd", "-Htf" }, { "rg", "-.g", "!.git", "--files" } }
+      or { { "fdfind", "-Htf", "--exclude", ".git" }, { "fd", "-Htf", "--exclude", ".git" }, { "rg", "-.g", "!.git", "--files" } }
```
Comment (delphinus) [inline lua/frecency/finder.lua:86]:
```suggestion
      or { { "fdfind", "-Htf", "-E", ".git" }, { "fd", "-Htf", "-E", ".git" }, { "rg", "-.g", "!.git", "--files" } }
```

**Review by delphinus — COMMENTED** [review]
I want to use shorthand options as all other ones are.

**Comment by matthewgrossman** [comment]
Committed your suggestions 👍

**Pull request merged**